ramls/marccat.raml

```diff
@@ -42,8 +42,16 @@ resourceTypes:
       responses:
         200:
           description: "Return a list of marc records and search metadata"
-        500:
-          description: "Internal server error"
+          400:
```

Comment:
This is broken indentation, and crashes the parser.

Reply:
right @dcrossleyau right, thanks.